plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMergeSink.java

```diff
@@ -365,7 +365,7 @@ private Slice writeMergeResult(Slice path, FileDeletion deletion)
             ParquetMetadata parquetMetadata = MetadataReader.readFooter(dataSource, Optional.empty());
             long rowCount = parquetMetadata.getBlocks().stream().map(BlockMetadata::rowCount).mapToLong(Long::longValue).sum();
             RoaringBitmapArray rowsRetained = new RoaringBitmapArray();
-            rowsRetained.addRange(0, rowCount);
+            rowsRetained.addRange(0, rowCount - 1);
```

Comment:
Please add javadoc comment to `io.trino.plugin.deltalake.delete.RoaringBitmapArray#addRange` that `rangeEnd` is inclusive